server/db.ts

```diff
@@ -1,50 +1,83 @@
-export const messages = [
-  {
-    id: "1",
-    content: "You on your way?",
-    createdAt: new Date(new Date("1-1-2019").getTime() - 60 * 1000 * 1000)
-  },
-  {
-    id: "2",
-    content: "Hey, it's me",
-    createdAt: new Date(new Date("1-1-2019").getTime() - 2 * 60 * 1000 * 1000)
-  },
-  {
-    id: "3",
-    content: "I should buy a boat",
-    createdAt: new Date(new Date("1-1-2019").getTime() - 24 * 60 * 1000 * 1000)
-  },
-  {
-    id: "4",
-    content: "This is wicked good ice cream.",
-    createdAt: new Date(
-      new Date("1-1-2019").getTime() - 14 * 24 * 60 * 1000 * 1000
-    )
-  }
-];
-export const chats = [
-  {
-    id: "1",
-    name: "Ethan Gonzalez",
-    picture: "https://randomuser.me/api/portraits/thumb/men/1.jpg",
-    lastMessage: "1"
-  },
-  {
-    id: "2",
-    name: "Bryan Wallace",
-    picture: "https://randomuser.me/api/portraits/thumb/men/2.jpg",
-    lastMessage: "2"
-  },
-  {
-    id: "3",
-    name: "Avery Stewart",
-    picture: "https://randomuser.me/api/portraits/thumb/women/1.jpg",
-    lastMessage: "3"
-  },
-  {
-    id: "4",
-    name: "Katie Peterson",
-    picture: "https://randomuser.me/api/portraits/thumb/women/2.jpg",
-    lastMessage: "4"
-  }
-];
+//creating type message and chat
+export type Message = {
+  id: string;
+  content: string;
+  createdAt: Date;
+};
+export type Chat = {
+  id: string;
+  name: string;
+  picture: string;
+  messages: string[];
```

Comment:
@wtrocki agreed and +1 for adding that context however, this is following the approach from the whatsapp clone tutorial where concepts are introduced iteratively 🙂

Reply:
thanks guys we are going to review our schema later on. I know it's not right approach but for now we are trying to follow tutorial so we won't get lost. I realize that it will be an extra work for us but I learn all this as I go and it's all relatively new to me.